clar2wasm/src/wasm_generator.rs

```diff
@@ -1686,7 +1686,7 @@ impl WasmGenerator {
                     builder.call(
                         self.module
                             .funcs
-                            .by_name("stdlib.stx_liquid_supply")
+                            .by_name("stx_liquid_supply")
```

Comment:
Sames as above. `stdlib.` prefix is now part of function present on Clarity standard library.